### PR TITLE
[Filter] Filter fixes

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes: AB#FILL_IN_YOUR_ISSUE_ID

## Checklist:

-   [ ] I have performed a self-review of my own code.
-   [ ] I have linked my DevOps task using the AB# tag.
-   [ ] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/Core/WorkSpace/src/Components/QuickFilter/FilterGroup/FilterGroupPopoverMenu.tsx [hunk 1]
```diff
@@ -118,3 +120,44 @@ export const FilterGroupPopoverMenu = ({
...
 };
+
+const List = styled.div`
+    max-height: 250px;
+    padding: 8px 8px;
+    overflow: scroll;
+    height: auto;
+`;
+
+interface VirtualListProps {
```
Comment (Noggling) [inline src/Core/WorkSpace/src/Components/QuickFilter/FilterGroup/FilterGroupPopoverMenu.tsx:131]: egen fil?

File: src/Core/WorkSpace/src/Components/QuickFilter/FilterQuickSearch.tsx [hunk 2]
```diff
@@ -31,22 +45,74 @@ export const FilterQuickSearch = (): JSX.Element => {
...
+
+interface MenuItem {
+    title: string;
+    onCLick: () => void;
+}
+interface SearchPickerDropdownProps {
+    menuItems: MenuItem[];
+}
+
+export function SearchPickerDropdown({ menuItems }: SearchPickerDropdownProps): JSX.Element {
```
Comment (Noggling) [inline src/Core/WorkSpace/src/Components/QuickFilter/FilterQuickSearch.tsx:87]: 1 component per fil ;)

File: src/modules/powerBI/src/Components/Filter/FilterItems/FilterItems.tsx [hunk 3]
```diff
@@ -66,17 +69,20 @@ export const FilterItems = ({
...
                 title={group.type}
+                hasActiveFilters={Boolean(activeFilters[group.type].length)}
+                controller={controller}
+                deselectAllValues={() => controller.deselectAllValues(group, filterValues[0])}
                 onSearch={handleOnSearchChange}
                 searchEnabled={group.filterVals.length > 7}
             />
             <CheckboxWrap ref={parentRef}>
-                <Checkbox
+                {/* <Checkbox
```
Comment (Noggling) [inline src/modules/powerBI/src/Components/Filter/FilterItems/FilterItems.tsx:79]: ??? utkomenter kode

Comment (Gustav-Eikaas) [reply src/modules/powerBI/src/Components/Filter/FilterItems/FilterItems.tsx:79]: Regner med det kommer kommentar på at de vil ha det inn igjen..

File: src/Core/WorkSpace/src/Components/QuickFilter/SearchPickerDropdown.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import { Menu } from '@equinor/eds-core-react';
+import { tokens } from '@equinor/eds-tokens';
+import { useRef, useState } from 'react';
+import { ClickableIcon } from '../../../../../packages/Components/Icon';
```
Comment (Noggling) [inline src/Core/WorkSpace/src/Components/QuickFilter/SearchPickerDropdown.tsx:4]: alias ??

File: src/modules/powerBI/src/Components/Filter/PopoverGroup/VirtualList.tsx [hunk 5]
```diff
@@ -1,7 +1,8 @@
+import { tokens } from '@equinor/eds-tokens';
+import { VirtualFilterItemCheckbox } from '@equinor/WorkSpace';
```
Comment (Noggling) [inline src/modules/powerBI/src/Components/Filter/PopoverGroup/VirtualList.tsx:2]: vet ikke om jeg liker dette. hvorfor ligger dette i @equinor/WorkSpace og ikke i filteret?

**Review by Noggling — APPROVED** [review]

**Review by Noggling — APPROVED** [review]

**Pull request merged**